### PR TITLE
Chapter 5: Filling in the layout

### PR DESCRIPTION
**Pull request by nguyenvantrung2015**
Chapter 5: Filling in the layout + i18n + check rubocop ok 
image

### CODE REVIEW

**Comment by nguyenvantrung2015** [comment]
ready

**Comment by nguyenvantrung2015** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -0,0 +1,98 @@
+@import "bootstrap-sprockets";
```
Comment (thanhhungchu95) [inline app/assets/stylesheets/custom.scss]: dùng nháy đơn trong file scss

**Comment by nguyenvantrung2015** [comment]
ready

**Pull request merged**